node/src/Pos/Wallet/Web/State/State.hs

```diff
@@ -280,8 +283,10 @@ removeNextUpdate = updateDisk A.RemoveNextUpdate
 testReset :: WebWalletModeDB ctx m => m ()
 testReset = updateDisk A.TestReset
 
-updateHistoryCache :: WebWalletModeDB ctx m => CId Wal -> [TxHistoryEntry] -> m ()
-updateHistoryCache cWalId = updateDisk . A.UpdateHistoryCache cWalId
+updateHistoryCache
+    :: WebWalletModeDB ctx m
+    => CId Wal -> HistoryCacheUpdate -> m ()
+updateHistoryCache = updateDisk ... A.UpdateHistoryCache
```

Comment:
Can we please avoid the `...` idiom.

Reply:
Why? 
All functions in this module carry semantics of applying `updateDisk` or `quieryDisk` to acid events with no regard to how many arguments these events contain. `...` seems to fit essentially here